modules/schedule/outputs.tf

```
@@ -0,0 +1,4 @@
+output "schedule_id" {
+  description = "The ID of the Opsgenie Schedule"
+  value       = try(opsgenie_schedule.default[*].id, null)
```

```suggestion
  value       = try(opsgenie_schedule.default[0].id, null)
```

[*] is the new splat operator (https://www.terraform.io/language/expressions/splat), it will return a list (or convert a single value to a list), and then `[*].id` will return a list of IDs (not a single ID as the output is intended to return).